cli/cli.go

```diff
@@ -145,13 +146,19 @@ func Run(args []string, tty terminal.Terminal) int {
 			}
 		}
 
-		var hostname string
-		if *connectName != "" {
-			hostname = *connectName
+		// Determine what name the server's certificate should match
+		var expectedNameInCertificate string
+		if *connectVerifyExpectedName != "" {
```

Comment:
Optional. I prefer a switch in these situations:

```
switch {
  case *connectVerifyExpectedName != "":
    expectedNameInCertificate = *connectVerifyExpectedName
  case *connectName != "":
    expectedNameInCertificate = *connectName
  default:
    expectedNameInCertificate = strings.Split(*connectTo, ":")[0]
}
```

Reply:
oooh. That's nice! done.